backend/core/views.py

```
@@ -29,38 +50,134 @@ def create(self, request, *args, **kwargs):
-        cards = Card.objects.all()
+        user = request.user
+        profile = get_object_or_404(Profile, user=user)
 
-        if not cards:
-            return Response({"error": "No Cards found"}, status=status.HTTP_404_NOT_FOUND)
-        
-        # pick a random card
-        # random_card = cards.order_by('?').first()
-        random_card = random.choice(cards)
+        # Get due review cards for the user
+        due_review_cards = ReviewCard.objects.filter(user=user, due=datetime.now(timezone.utc))
```

Ruff

`ReviewCard` may be undefined, or defined from star imports

(F405)